src/ThirdParty/Gitlab/Adapter/GitLabIssueTracker.php

```diff
@@ -190,7 +196,18 @@ public function getComments($id)
             $this->client->api('issues')->show($this->getCurrentProject()->id, $id)
         );
 
-        return $issue->showComments();
+        $comments = [];
+        $issue->showComments();
```

Comment:
Never mind, I see when setting the data on the model, it only sets the data that is available in the properties